### ISSUE TITLE
[Bug] CSS import fails parsing in some cases

### ISSUE THREAD

**Issue by phoenisx** [willofindie/vscode-cssvar#97]
**Describe the bug**
CSS imports in a project using vite/webpack fails to get parsed by extension

**To Reproduce**

```css
@import 'modern-normalize/modern-normalize.css';

:root {
  --foo: red;
}
```

The above CSS will fail to get parsed.

**Expected behavior**
Either make this functional or gracefully ignore such imports, as such imports
can be defined in Extension settings as well.

**Comment by TommySorensen** [comment]
Also fails with something like this
```css
@custom-selector :--root :root, :after, :before;

@layer variables {
  @layer colors {
    :--root {
      --something: rgba(17, 17, 17, 1);
    }
  }
```

**Comment by phoenisx** [comment]
This is fixed in `v2.6.0`.

@TommySorensen I couldn't replicate your issue on my system. I tried testing this in this example: https://github.com/willofindie/vscode-cssvar/blob/main/examples/basics/theme.css#L24, but it was working fine for me.


Can u try to replicate this into a small repro, and open a new issue, if it's still not fixed? Thanks
